src/NexusMods.Collections/CollectionCreator.cs

```diff
@@ -0,0 +1,159 @@
+using DynamicData.Kernel;
+using NexusMods.Abstractions.Collections.Json;
+using NexusMods.Abstractions.Library.Models;
+using NexusMods.Abstractions.Loadouts;
+using NexusMods.Abstractions.MnemonicDB.Attributes;
+using NexusMods.Abstractions.NexusModsLibrary;
+using NexusMods.Abstractions.NexusModsLibrary.Models;
+using NexusMods.Abstractions.NexusWebApi;
+using NexusMods.Abstractions.NexusWebApi.Types;
+using NexusMods.MnemonicDB.Abstractions;
+using NexusMods.Paths;
+using CollectionMod = NexusMods.Abstractions.Collections.Json.Mod;
+
+namespace NexusMods.Collections;
+
+public static class CollectionCreator
+{
+    public static (CollectionRoot, CollectionMetadata.New, CollectionRevisionMetadata.New) CreateCollection(
```

Comment:
Similar to class, please add comments to explain what this supposed to do, as CreateCollection is ambiguous and doesn't define the scope of the operation well.